### DISCUSSION TITLE
How to install Matplotlib and pandas?

### DISCUSSION THREAD

**Discussion by rucikas** [ish-app/ish#1567] [Q&A]
Hey all,
Can anyone have idea how to install them?
I tried with pip3, but after about 24hours of waiting still nothing…

**Comment by magmaus3** [comment]
Doing that with pip is very slow, maybe you should try using other methods

**Comment by drews54** [comment]
As mentioned above, you could try searching for needed Python modules in apk repositories instead of PyPi. For Python 3 they are usually prepended with `py3-*`. In your case, try `apk add py3-matplotlib py3-pandas`

**Reply by loda91** [reply]
Not working `apk add py3-matplotlib py3-pandas`

`apk add py3-matplotlib` work

`apk add py3-pandas` not working also

**Reply by drews54** [reply]
> Not working `apk add py3-matplotlib py3-pandas`
> 
> `apk add py3-matplotlib` work
> 
> `apk add py3-pandas` not working also

Would you mind sharing an error message given by apk? I'm unable to reproduce it, as in – it installs just fine.

**Reply by jhunley** [reply]
I have the same issue on the current AppStore version:

Cerebrus:~# apk add py3-pandas
fetch http://apk.ish.app/v3.12-2021-06-25/main/x86/APKINDEX.tar.gz
fetch http://apk.ish.app/v3.12-2021-06-24/community/x86/APKINDEX.tar.gz
ERROR: unable to select packages:
  py3-pandas (no such package):
    required by: world[py3-pandas]

It just is not in the repository. // $ apk list | grep pandas // comes up empty.
